scss/widgets/embedded-posts.scss

```diff
@@ -17,7 +17,7 @@
   }
 }
 
-section.embedded-posts.top.topic-body.offset2:not(.dc-component) {
+section.embedded-posts.top.topic-body:not(.dc-component) {
```

Comment:
Seems like there were a class that no longer is there @orlando that's why the bug came up.